OrbitGl/App.cpp

```diff
@@ -552,17 +551,40 @@ ErrorMessageOr<void> OrbitApp::OnLoadPreset(const std::string& filename) {
 void OrbitApp::LoadPreset(const std::shared_ptr<PresetFile>& preset) {
   const int32_t selected_process_id = GetSelectedProcessId();
   const ProcessData* selected_process = data_manager_->GetProcessByPid(selected_process_id);
-  const std::string& process_full_path = preset->preset_info().process_full_path();
-  if (selected_process != nullptr && selected_process->full_path() == process_full_path) {
-    // In case we already have the correct process selected
-    GOrbitApp->LoadModulesFromPreset(GetSelectedProcess(), preset);
-    return;
+  if (selected_process != nullptr) {
+    LoadModulesFromPreset(GetSelectedProcess(), preset);
+  } else {
+    SendErrorToUi("Preset loading failed", "Process is not selected");
```

Comment:
This feels a bit wrong. Since we knew that upfront we could have told the user more clearly by disabling the preset-list when no process is selected (and adding a tooltip). But since this is not a regression I'd suggest we create a bug for that and leave it like it is.

Reply:
Thanks for the suggestion! Created a bug for it http://b/167380800